update.sh

```diff
@@ -12,17 +12,29 @@ versions=( "${versions[@]%/}" )
 travisEnv=
 for version in "${versions[@]}"; do
 	travisEnv='\n  - VERSION='"$version$travisEnv"
-	
-	repoBase="http://packages.elasticsearch.org/elasticsearch/$version/debian"
-	if [ "${version%%.*}" -ge 2 ]; then
-		repoBase="http://packages.elasticsearch.org/elasticsearch/${version%%.*}.x/debian"
+
+	rcVersion="${version%-rc}"
```

Comment:
In the other places we've used `-rc` to mean "pre-release" generally, which IMO is reasonably sane (since most upstreams don't seem to treat different types of pre-releases much differently).